server/views/courses/index.njk

```diff
@@ -7,13 +7,8 @@
 
   <div role="list">
     {% for course in courses %}
-      {{ indexCourseSummary(course) }}
-
-      {% if loop.index < courses.length %}
-        <div class="govuk-grid-row">
-          <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-6">
-        </div>
-      {% endif %}
+      {% set isLastInList = loop.index >= courses.length %}
```

Comment:
FYI (you're probably aware of this) but `loop.index0` is also a thing in Nunjucks if you want the index to start at 0, but I think this is more readable anyway

Reply:
Ah yes, I'm sure you've mentioned that before. I'm easy - I think this should be fine as long as there's an explanation in the commit if someone is thinking the maths is off, and agree it's more readable